libs/payments/ui/src/lib/client/components/Header/index.tsx

```diff
@@ -0,0 +1,372 @@
+            >
+              <Image
+                src={helpIcon}
+                aria-label="Help"
+                alt="Help"
+                role="img"
+                className="w-5 text-violet-900"
+              />
+            </LinkExternal>
+          </Localized>
+          {/** Bento Menu */}
+          <div
+            className="relative self-center flex mx-2"
+            ref={bentoMenuInsideRef}
+          >
+            <Localized id="next-bento-menu">
+              <button
+                onClick={toggleBentoMenuRevealed}
+                title="Mozilla products"
+                aria-label="Mozilla products"
+                aria-expanded={!!isBentoMenuRevealed}
+                aria-haspopup="menu"
+                className="rounded p-2 mx-2 border-transparent hover:bg-grey-100 transition-standard desktop:mx-8"
+              >
+                <Image
+                  src={bentoIcon}
+                  alt="Mozilla Logo"
+                  className="w-5 text-violet-900"
+                />
+              </button>
+            </Localized>
+
+            {/** Bento Dropdown */}
+            {isBentoMenuRevealed && (
+              <div
+                className="w-full h-full fixed top-0 start-0 bg-white z-10
+                      mobileLandscape:h-auto mobileLandscape:drop-down-menu mobileLandscape:top-10 mobileLandscape:-start-52 desktop:-start-50"
+              >
+                <div className="flex flex-wrap">
+                  <div className="flex w-full py-4 gap-2 items-center flex-col tablet:w-auto tablet:relative">
+                    <button onClick={() => setBentoMenuRevealed(false)}>
+                      <Image
+                        src={closeIcon}
+                        alt="Close"
+                        width="16"
+                        height="16"
+                        className="absolute top-5 end-5 mobileLandscape:hidden fill-current"
+                      />
+                    </button>
+                    <div className="mt-12 px-8 text-center mobileLandscape:mt-0">
+                      <Localized id="next-bento-menu-tagline">
+                        <h2>
+                          More products from Mozilla that protect your privacy
+                        </h2>
+                      </Localized>
+                    </div>
+                    <div className="w-full text-xs">
+                      <ul className="list-none">
+                        <li>
+                          <LinkExternal
+                            href={links.desktop}
+                            className="block p-2 ps-6 hover:bg-grey-100"
+                          >
+                            <div className={iconClassNames}>
+                              <Image src={desktopIcon} alt="" />
+                            </div>
+                            <Localized id="next-bento-menu-firefox-desktop">
+                              Firefox Browser for Desktop
```

Comment:
Aah, Reino recently reminded me that we should be wrapping what we want `Localized` to translate in a HTML element, so: 
```
<Localized id="next-bento-menu-firefox-desktop">
  <span>Firefox Browser for Desktop</span>
</Localized>
```

I've been cleaning these up as I go along, but should probably make a polish PR.

Reply:
sounds good!